.env.example

```diff
@@ -58,3 +58,13 @@ TRIGGER_CONFIG='{"endpoints":{"n8n":{"url":"http://n8n:5678/webhook/1759ab31-e34
 # Format: http://n8n-host:port/webhook/your-webhook-path
 # Default in docker: http://n8n:5678/webhook/irc
 N8N_WEBHOOK=http://n8n:5678/webhook/irc
+
+# Flood Protection
+# Comma-separated list of channels where flood protection is enabled
+FLOOD_PROTECTED_CHANNELS=
+
+# Maximum lines to send before using paste service (default: 3)
+MAX_LINES_BEFORE_PASTING=3
+
+# Curl template for paste service. {{filename}} will be replaced with the temporary file path
+PASTE_CURL_TEMPLATE=curl -s -F "file=@{{filename}}" https://ix.io
```

Comment:
I feel like this should be more like:

`PASTE_URL=https://ix.io` 

Since the curl command is quite universal to filehosts, I feel the curl command and the rest of it (except the url) should be in the code normally. What are your thoughts?

Reply:
Each pastebin has its own command. it is not standardized.
https://github.com/skorokithakis/pastebinit/tree/master/pastebin.d

Can't find a better proof but sometimes is json, sometimes it is a form, sometimes is raw, sometimes it responds with json, sometimes just the url. So I would rather leave this flexible.